packages/wrap/src/transforms/WrapQuery.ts

```diff
@@ -87,11 +91,18 @@ export default class WrapQuery<TContext = Record<string, any>>
       const path = [...this.path];
       while (path.length > 1) {
         const next = path.shift()!;
+        if (isPrototypePollutingKey(next)) {
```

Comment:
what's the case here? I mean, is it passed from a user? i mean, i don't understand how this could happen?

Reply:
`path` comes from the user input in the constructor.